api/chat_main_handler.go

```diff
@@ -13,7 +13,6 @@ import (
 	"strings"
 
 	uuid "github.com/iris-contrib/go.uuid"
-	"github.com/samber/lo"
 	openai "github.com/sashabaranov/go-openai"
 	"github.com/swuecho/chatgpt_backend/sqlc_queries"
 
```

Comment:
:robot: OpenAI

This patch removes the import of `github.com/samber/lo`. 

If this package is not used anywhere in the code, then it's safe to remove it. However, if it is used elsewhere, removing it here will cause a compilation error.

Please confirm that this package is not used anywhere else in the codebase before merging this patch.

Also, please update the comments on the pull request to reflect this change.

<!-- This is an auto-generated comment by OpenAI -->